sklearn/model_selection/tests/test_search.py

```diff
@@ -590,9 +602,15 @@ def test_X_as_list():
 
     clf = CheckingClassifier(check_X=lambda x: isinstance(x, list))
     cv = KFold(n_splits=3)
-    grid_search = GridSearchCV(clf, {'foo_param': [1, 2, 3]}, cv=cv)
-    grid_search.fit(X.tolist(), y).score(X, y)
-    assert_true(hasattr(grid_search, "cv_results_"))
+
+    for scoring in (None, 'accuracy', ('accuracy', ),
```

Comment:
I don't really see why this is a test you would expect to be affected by this PR. It seems like unnecessary verbosity added to the tests. Maybe I've just not thought it through.